LeaderboardBackend/Models/Requests/BanRequests.cs

```diff
@@ -40,6 +40,7 @@ public record CreateLeaderboardBanRequest
 	[Required]
 	public string Reason { get; set; } = null!;
 
+	// NOTE: Should this be [Required]? - Ero
```

Comment:
Yeah, this should be Required now, since we've got separate leaderboard ban request and site ban request objects now.